opencxl/cxl/transport/transaction.py

```diff
@@ -1016,7 +1016,7 @@ def create(uqid: int, opcode: CXL_CACHE_D2HRSP_OPCODE) -> "CxlCacheCacheD2HRspPa
         packet = CxlCacheCacheD2HRspPacket()
         packet.system_header.payload_type = PAYLOAD_TYPE.CXL_CACHE
         packet.system_header.payload_length = len(packet)
-        packet.cxl_cache_header.msg_class = CXL_CACHE_MSG_CLASS.D2H_REQ
+        packet.cxl_cache_header.msg_class = CXL_CACHE_MSG_CLASS.D2H_RSP
```

Comment:
Looks like you forgot to rebase/merge. This change has already been merged in on another PR.